iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/BusBreakerVoltageLevel.java

```diff
@@ -353,7 +353,7 @@ public void edgeRemoved(int e, SwitchImpl obj) {
             }
 
             @Override
-            public void allEdgesRemoved() {
+            public void allEdgesRemoved(Collection<SwitchImpl> obj) {
                 invalidateCache();
```

Comment:
Sorry I'm missing something here, why didn't we remove the switches from the network index in bus breaker view?

Reply:
We do remove the switches from the network index in bus breaker view but it is not done in the graph listener (I don't actually remember why)